### ISSUE TITLE
Divison by zero when useLumpedPressure is enabled in pipe model

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#233]
**Reported by anton&#46;sodja on 19 Nov 2009 10:24 UTC**
When useLumpedPressure is enabled (under `Advanced' tab in settings dialog of DynamicPipe model) the following error is reported at the begining of the simulation run:
```
dymosim started
Model: PipeLumpedPressureBug
... "dsin.txt" loading (dymosim input file)
The following error was detected at time: 0
 
Model error - division by zero: (0) / (0) = (0) / (0)
```
This error is reported regardless of the 'Medium' and 'FlowModel' settings.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/233

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 6 Jul 2012 09:28 UTC**
#711 was reported as duplicate of this.

**Comment by modelica-trac-importer** [comment]
**Modified by rfranke on 27 Dec 2012 12:26 UTC**

**Comment by modelica-trac-importer** [comment]
**Changelog removed by rfranke on 27 Dec 2012 12:26 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by rfranke on 27 Dec 2012 12:28 UTC**
This has been fixed in 37218482198b481f61456887b9969d5dcfebc843. See also comments in #711.

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 17 Jan 2013 15:24 UTC**
